cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1287,6 +1287,14 @@
     </description>
   </property>
 
+  <property>
+    <name>data.storage.properties.gcp-spanner.appspec.reduction</name>
+    <value>true</value>
```

Comment:
Since this is configurable. 

Let's say first it's enabled.  Some AppSpec is written to DB using the new adaptors. 
Now we disable this. 
It will use the default GSON `ApplicationSpecificationAdapter` only.  Will that not cause any issue  ??